app/assets/stylesheets/provision.scss

```diff
@@ -0,0 +1,3 @@
+.row-button {
+  margin-bottom:15px;
```

Comment:
I know it's a little overkill to add a stylesheet just for this but we might have more to add in auto_provision_all form and I could change this to `application.scss` in case there might be changes in other areas of the plugin.
I don't mind removing this if that's preferred.